src/stories/FormInput.stories.tsx

```
@@ -0,0 +1,40 @@
+import { Meta } from '@storybook/react';
+
+const meta: Meta<typeof FormInput> = {
+  title: 'FormInput',
+  component: FormInput,
+  tags: ['autodocs'],
+};
+
+export default meta;
+
+export const Default = (args: FormInputElementProps) => <FormInput {...args} />;
+export const Error = (args: FormInputElementProps) => <FormInput {...args} />;
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)